### PR TITLE
docs: fix media library usage

### LINKED ISSUE #2954: Docs+Blog: fix external media library usage in blog and update docs
Prior issue: https://github.com/netlify/netlify-cms/issues/2950.

When using `netlify-cms-app` it is not clear that one needs to explicitly register the media library.
The information is available here:
https://www.netlifycms.org/blog/2019/07/netlify-cms-gatsby-plugin-4-0-0#using-media-libraries-with-netlify-cms-app

But the code example is wrong.
It should be:

```js
import uploadcare from 'netlify-cms-media-library-uploadcare';

CMS.registerMediaLibrary(uploadcare);
```

And not:

```js
import uploadcare from 'netlify-cms-media-library-uploadcare'

CMS.registerMediaLibrary('uploadcare', uploadcare)
```
See here: https://github.com/netlify/netlify-cms/blob/237a5b1aa65098f9c5b3768a306633a01fa25534/packages/netlify-cms/src/media-libraries.js#L5

We should add a note to it in the documentation as well.

### PR DESCRIPTION
**Pull request by tiuweehan**
Fixes #2954 

I'm not too sure if I have included the note for `netlify-cms-app` users in the most appropriate place.

### CODE REVIEW

**Comment by erezrokah** [comment]
@tiuweehan thank you for taking the initiative on this issue. Will review shortly

File: website/content/blog/netlify-cms-gatsby-plugin-4-0-0.md [hunk 1]
```diff
@@ -96,15 +96,15 @@ The Netlify CMS media library extensions for Cloudinary and Uploadcare are not i
...
 // here for example purposes.
-import uploadcare from 'netlify-cms-media-library-uploadcare'
-import cloudinary from 'netlify-cms-media-library-cloudinary'
+import uploadcare from 'netlify-cms-media-library-uploadcare';
+import cloudinary from 'netlify-cms-media-library-cloudinary';
 
-CMS.registerMediaLibrary('uploadcare', uploadcare)
-CMS.registerMediaLibrary('cloudinary', cloudinary)
+CMS.registerMediaLibrary(uploadcare);
+CMS.registerMediaLibrary('cloudinary', cloudinary);
```
Comment (erezrokah) [inline website/content/blog/netlify-cms-gatsby-plugin-4-0-0.md]: I think this should be `CMS.registerMediaLibrary(cloudinary);`

Comment (tiuweehan) [reply website/content/blog/netlify-cms-gatsby-plugin-4-0-0.md]: @erezrokah Thanks for helping to review! I have fixed the issue.

Comment (fabianrios) [reply website/content/blog/netlify-cms-gatsby-plugin-4-0-0.md]: with all the decap rebranding, this is still to be written somewhere, which file is supposed to have this, any link to updated documentation?

**Pull request merged**